e2e/wdio/core/tests/action-bar.e2e-spec.ts

```diff
@@ -0,0 +1,307 @@
+import { ActionBarPo } from '../pages/action-bar.po';
+import {
+    addIsActiveClass,
+    checkElementScreenshot,
+    click,
+    getElementArrayLength,
+    mouseHoverElement,
+    refreshPage,
+    saveElementScreenshot,
+    scrollIntoView,
+    waitForElDisplayed
+} from '../../driver/wdio';
+import {
+    backButtonExample,
+    backButtonFocusState,
+    backButtonHoverState,
+    cancelButtonFocusState,
+    cancelButtonHoverState,
+    contextualMenuExample,
+    longTitleExample,
+    menuButtonFocusState,
+    menuButtonHoverState,
+    menuItemActiveState,
+    menuItemFocusState,
+    menuItemHoverState,
+    mobileViewExample,
+    noBackButtonExample,
+    saveButtonFocusState,
+    saveButtonHoverState
+} from '../fixtures/testData/action-bar-tags';
+
+describe('Action Bar Test Suite', function() {
+    const actionBarPage = new ActionBarPo();
+    const {
+        actionBarBackButtonBackButton,
+        actionBarBackButtonCancelButton,
+        actionBarBackButtonSaveButton,
+        actionBarLongPageTitle,
+        actionBarLongPageTitleBackButton,
+        actionBarLongPageTitleCancelButton,
+        actionBarLongPageTitleSaveButton,
+        actionBarNoBackButton,
+        actionBarNoBackButtonCancelButton,
+        actionBarNoBackButtonSaveButton,
+        actionBarMobileViewBackButton,
+        actionBarMobileViewMenuButton,
+        actionBarMobileViewOptionList,
+        actionBarMobileViewOptionListItem,
+        actionBarContextualMenuButton,
+        actionBarContextualMenuOptionList,
+        actionBarContextualMenuOptionListItem
+    } = new ActionBarPo();
+
+    beforeAll(() => {
+        actionBarPage.open();
+        waitForElDisplayed(actionBarBackButtonBackButton);
+    }, 1);
+
+    describe('verify Back button example', function() {
+        it('should check back button hover state', () => {
+            checkButtonHoverState(actionBarBackButtonBackButton, backButtonExample + backButtonHoverState, 'Back');
```

Comment:
let's make the button names variables too, you use them multiple times and they could change. 'back', 'cancel' etc.